### DISCUSSION TITLE
[RESOLU] Erreurs lors de la migration

### DISCUSSION THREAD

**Discussion by Moustash** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#113] [Q&A] [answered]
Bonjour,

Je suis actuellement confronté à des erreurs lors de la migration initiales des data. 

A noter que j'ai bien des données qui ont été importées dans ma base.

Je rencontre aussi des erreurs dans la requête numéro 11, et la numéro 37. 

![image](https://user-images.githubusercontent.com/38319075/219399651-e491ac92-47d3-47dd-8756-28047637d94a.png)

![image](https://user-images.githubusercontent.com/38319075/219421872-3772fb40-905b-4979-b3e4-36020145ee6d.png)

![image](https://user-images.githubusercontent.com/38319075/219422026-8582caa3-eb24-4aab-b323-2c3d526dae9a.png)

![image](https://user-images.githubusercontent.com/38319075/219422263-7fcd1a05-43aa-4dce-b170-b7b80ed8766e.png)

![image](https://user-images.githubusercontent.com/38319075/219399833-05c4409a-0cf3-4a81-ab9b-fb79b99498a6.png)

**Comment by Laikker** [comment]
Hello,
J'ai le même problème que toi quand j'utilise MySQL par contre c'est OK avec MariaDB même sous windows
Avec wamp par default, MariaDB tourne sur le port 3307

**Reply by Moustash** [reply]
Hello, merci de ta réponse.

J'ai aussi rencontré le probleme avec mariadb. Je suis en train de mettre en place un WLS pour être ISO linux sur mon projet. Merci !

**Reply by Moustash** [reply]
Cela dit, je rencontre un nouveau soucis lié cette fois ci à QC-CORE qui m'empêche de me connecter au serveur.

![image](https://user-images.githubusercontent.com/38319075/219517541-20bad5b5-a209-49bf-860d-3e5a4beddd2e.png)

**Comment by Moustash** [comment, marked as answer]
RESOLU : Il semblerait que l'erreur SQL soit "normale" et n'était en fait pas la cause de mon probleme. En WSL, il me faut utiliser mon IP publique et non mon IP localhost pour acceder à fivem depuis windows.
